### PR TITLE
 Switch from post to get for "list" and "listActiveCheckIns"

### PR DESCRIPTION
**Pull request by pbek**
See https://github.com/studo-app/campus-qr/commit/8fe8c63b1dc0ff4ec493bb816a2666bf59f6c16b#r42883361

### CODE REVIEW

File: src/serverMain/kotlin/com/studo/campusqr/Server.kt [hunk 1]
```diff
@@ -126,8 +126,8 @@ suspend fun main() {
         }
       }
       route("report") {
-        post("list") { call.getAuthenticatedCall()?.returnReportData() }
-        post("listActiveCheckIns") { call.getAuthenticatedCall()?.listAllActiveCheckIns() }
+        get("list") { call.getAuthenticatedCall()?.returnReportData() }
```
Comment (functionaldude) [inline src/serverMain/kotlin/com/studo/campusqr/Server.kt:129]: this endpoint is used in the moderation frontend so you have to adjust the XHR call there.

**Review by functionaldude — CHANGES_REQUESTED** [review]
When you switch from `POST` to `GET` you have to use `call.parameters` instead of `receiveJsonStringMap()` because a `GET` has no request body.

**Comment by pbek** [comment]
We'll look in to that on Monday.

**Comment by pbek** [comment]
I'll create an issue meanwhile: https://github.com/studo-app/campus-qr/issues/40

**Pull request closed without merging**